src/components/info-general-step/InfoGeneralStep.tsx

```
@@ -0,0 +1,111 @@
+import interestImg from '~/assets/img/tutor-home-page/become-tutor/study-category.svg'
+import languageImg from '~/assets/img/tutor-home-page/become-tutor/languages.svg'
+import {
+  tutorStepLabels,
+  studentStepLabels,
+  initialValues
+} from '~/components/user-steps-wrapper/constants'
+import { styles } from './InfoGeneralStep.styles'
+import ConfirmDialog from '~/components/confirm-dialog/ConfirmDialog'
+
+
+const UserStepsModal = ({ role = 'tutor' }) => {
```

ESLint

[error] Delete `⏎`

(prettier/prettier)